templates/generic.py

```diff
@@ -26,9 +27,10 @@ class GenericTemplate:
     def __init__(self):
         self.template = copy(template['value'])
         self.elements = []
-    def add_element(self, title='', item_url='', image_url='', subtitle='', buttons=[]):
+    def add_element(self, title='', description='', item_url='', image_url='', subtitle='', buttons=[]):
         element = {}
         element['title'] = title[:TITLE_CHARACTER_LIMIT]
+        element['description'] = description[:DESCRIPTION_CHARACTER_LIMIT]
```

Comment:
So, messenger has some templates that it supports. I've created classes mapping those so that you only have to create a object with the required data, and the rest is taken care of (formatting the data as you see while interacting with the bot). The Generic Template is here: https://developers.facebook.com/docs/messenger-platform/send-api-reference/generic-template
As we are using third-party APIs for data, most of them are free to use but require us to add something like 'Powered by NewsAPI'. This check ensures that it's not our data to show, and we're just serving the data. This helps them gain audience. See the book / weather modules for reference.